internal/v2/tls_config_store/tls_config_store.go

```diff
@@ -124,6 +124,7 @@ func GetTLSConfigurationForClient(serverHostname string, cstream s2av2pb.S2AServ
 		ClientSessionCache:    nil,
 		MinVersion:            minVersion,
 		MaxVersion:            maxVersion,
+		NextProtos:            []string{"h2"},
```

Comment:
Please update the unit tests to check that NextProtos is what we expect.

Reply:
Will do!